### PR TITLE
add realtime statistic stdout and add cmake install

### PR DESCRIPTION
**Pull request by soyeonBak**

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -65,3 +65,7 @@ endif()
 if(RE2_LIB)
   target_link_libraries(regexbench ${RE2_LIB})
 endif()
+
+file(GLOB RB_HDR_SRCS "*.h")
```
Comment (kwanglink) [inline src/CMakeLists.txt]: Isn't it better to limit the number of include files to be open public?

File: src/regexbench.h [hunk 2]
```diff
@@ -48,13 +49,23 @@ namespace regexbench {
...
-  MatchResult() : nmatches(0), nmatched_pkts(0) {}
+  MatchResult() : stop(false) {}
 
   struct timeval udiff;
   struct timeval sdiff;
-  size_t nmatches;
-  size_t nmatched_pkts;
+  struct ResultInfo cur;
+  struct ResultInfo old;
+  bool stop;
```
Comment (kwanglink) [inline src/regexbench.h]: How about declaring this as `std::atomic_bool` and use corresponding API's where it's being used.

File: src/CMakeLists.txt (hunk 1 above)
Comment (soyeonBak) [reply src/CMakeLists.txt]: I'll limit it to what I need.

File: src/regexbench.h (hunk 2 above)
Comment (soyeonBak) [reply src/regexbench.h]: I'll try to apply it.

**Pull request merged**